lava_test_plans/testcases/master/template-android-xts.jinja2

```diff
@@ -0,0 +1,38 @@
+{% extends "testcases/master/template-master.jinja2" %}
+
+{% set xts_test_params = xts_test_params|default("xts-not-work") %}
+{## the test_name is a part of the job name ##}
+{% set test_name = test_name|default("xts") %}
+{## the test_suite_name is a part of the path of the lava result ##}
+{% set xts_test_suite_name = xts_test_suite_name|default(test_name) %}
+{% set xts_test_path = xts_test_path|default("xts-path") %}
+{% set xts_pkg_name = xts_pkg_name|default("android-xts.zip") %}
```

Comment:
looks like the path and zip file is more or less the same... maybe drop one variable?

Reply:
let's keep two variables here, the zip file name is not always like android-cts.zip, there are other places do the renaming for the current setup, maybe some time later the workaround there could be dropped